### DISCUSSION TITLE
Exclude transactions/account from in and out report

### DISCUSSION THREAD

**Discussion by Skoshche** [firefly-iii/firefly-iii#8487] [Firefly III]
I have a savings account which I get monthly deposits into automatically, while I would like to track how much I have in this account I would prefer not to have the transactions count for how much I am actually making/receiving, is there a way to exclude the transactions or an entire account not only from net worth but also in/out calculations?

**Comment by JC5** [comment]
You can exclude the account from reports, but other charts and bars will always include it.

**Comment by diamant-x** [comment]
I'm actually interested in something like that.

My scenario is, I'm investing in short term fixed rate bonds. (3 months due). If i ran the standard financial report. Every time I invest I see a huge withdrawal, and then when I receive it back I see a huge deposit (plus a smaller one for interests). While at core both transactions make sense, it distorts a lot my ability to read the transaction reports and understand 'new' money being spent and received.

I'm tracking them as liabilities (I'm owed money), so I guess a way to 'exclude from reporting' specific transactions would be a possible solution, or flag the 'opening' and 'closing' transactions from liabilities to not show.

- Any other ideas how to improve readibility of transactions when handling 'secure' short-term investments?

**Comment by diamant-x** [comment]
Answering to OP:

> I have a savings account which I get monthly deposits into automatically

If those deposits are'existing' money you are moving from your 'daily account' into a 'savings account' then just track both accounts (as [asset accounts](https://docs.firefly-iii.org/references/firefly-iii/account-types/#asset-accounts)) in FF3 and mark them as transfers. Transfer are tracked separate from in/out money.

![image](https://github.com/firefly-iii/firefly-iii/assets/7346338/9eb8dcc1-ca57-40f0-8ef3-1a89d3858d21)


If those deposits are 'benefits/interests' you are receiving, I would understand they are correctly flagged as new deposits and would need more context to understand why would you would like not to see them.

**Reply by Skoshche** [reply]
So i receive say 800$/mo from personal benefit stuff. I keep it in a side savings account and don't like to count it as money I can actively use, since its for college payments. I would like to still keep track of how much is in the bank/when it gets deposited/rare occasions i take a bit out from it, but would like it to not count as "money i received in the month" on the dashboard as I count my normal paycheck/money for bills and food etc. 

TLDR, I would like to track it but not count it as money I have received that I am able to use. Having it counted in the in/out statistic makes it so I am always in the positive for money in in/out instead of an accurate depiction of how much spending money I have received/spent.

**Reply by diamant-x** [reply]
I guess the closest option could be to flag that saving account as 'not count as net worth', but then I guess as developer replied above, deposits would still appear in other places.

![image](https://github.com/firefly-iii/firefly-iii/assets/7346338/f9255901-846c-4a27-9acc-57f426f3abdb)

That's as close as it gets I would say. End of the day is money you didn't had before and now you do, so is a deposit.
You could remove it from net worth the balances as mentioned above, or create a piggy bank to add those so that its clear to FF3 that such money has a specific purpose.
